schema_registry_test.go

```diff
@@ -287,21 +288,36 @@ func TestSchemaRegistryClientClass(t *testing.T) {
 		})
 		assert.NotNil(t, client)
 
-		// Create a schema and send it to the registry.
+		// Create first schema and send it to the registry.
 		createSchema := client.Get("createSchema").Export().(func(sobek.FunctionCall) sobek.Value)
-		newSchema := createSchema(sobek.FunctionCall{
+		newSchema1 := createSchema(sobek.FunctionCall{
 			Arguments: []sobek.Value{
 				test.module.vu.Runtime().ToValue(
 					map[string]interface{}{
 						"subject":    "test-subject",
-						"schema":     avroSchema,
+						"schema":     avroSchema1,
 						"schemaType": srclient.Avro,
 					},
 				),
 			},
 		}).Export().(*Schema)
-		assert.Equal(t, "test-subject", newSchema.Subject)
-		assert.Equal(t, 0, newSchema.Version)
```

Comment:
fyi: the first version of a schema should have the version `1` (not `0`)

Reply:
When we use the `client.CreateSchema` function in our code, it internally calls the `client.GetSchema` function from the `srclient` library. This function interacts with the Schema Registry API endpoint [`GET /schemas/ids/{id}`](https://docs.confluent.io/platform/current/schema-registry/develop/api.html#get--schemas-ids-int-%20id). According to the API documentation, this endpoint returns the schema associated with a specific ID but **does not include the version information** in its response. As a result, the version defaults to `0` when you retrieve the schema using these functions.

To obtain the version information of the schema, you should use the `getSchema` wrapper function like below. This function calls a different endpoint—such as `GET /subjects/{subject}/versions/{version}` or `GET /subjects/{subject}/versions/latest`—which provides both the schema and its version information. By using `getSchema`, you ensure that you retrieve all the necessary details about the schema, including its version.

(We can add this as a comment here)